Automattic/wp-calypso
# Marketing: Fix broken images in Marketing Tools

#### Changes proposed in this Pull Request

This PR adds the images that appear to be broken in Marketing Tools.

#### Testing instructions



1. Go to http://calypso.localhost:3000/marketing/tools/[site]
2. Check if images are no longer broken

cc @stephanethomas
